src/feed.js

```diff
@@ -352,20 +364,30 @@ class Feed {
           if (author.email && author.name) {
             item.push({ author: author.email + ' (' + author.name + ')' })
             return true
-          } else {
-            return false
+          } else if (author.name) {
+            item.push({ 'dc:creator': author.name  });
+            return true;
           }
+          return false;
         })
       }
 
-      if(item.image) {
-        item.push({ enclosure: [{ _attr: { url: entry.image } }] });
+      if(entry.image) {
+        var type = entry.image.split('.').pop().replace('jpg', 'jpeg');
```

Comment:
Please use `let`/`const` instead of `var`.
Also, this method to get the extension works only if the links are clean (eg: no querystring nor hashstring).
It wont work for something like this: http://my.image.server/bla.jpg?key=foobar. You may want to use this instead: [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) (compatibility here https://caniuse.com/#feat=url)

Reply:
this worked for me
```js
if (entry.image) {
      const type = new URL(entry.image).pathname.split('.').slice(-1)[0]
      item.enclosure = { _attributes: { url: entry.image, length: 0, type: `image/${type}` } }
    }
```